nitro-protocol/config/jest/jest.config.js

```diff
@@ -18,7 +18,7 @@ module.exports = {
   rootDir: root,
   collectCoverageFrom: ['**/*.{js,jsx,ts,tsx}'],
   reporters: ['default'],
-  testMatch: ['<rootDir>/test/src/**/?(*.)test.ts?(x)'],
+  testMatch: ['<rootDir>/test/src/**/?(*.)test.ts?(x)', '<rootDir>/test//?(*.)test.ts?(x)'],
```

Comment:
Do you know why the double `/` is needed in `'<rootDir>/test//?(*.)test.ts?(x)'`?

Reply:
It is not needed, it is a typo :-P. It turns out that it still works, though -- you can try it out here https://www.digitalocean.com/community/tools/glob .